plugins/catalog-unprocessed-entities/README.md

```diff
@@ -20,6 +20,30 @@ import { CatalogUnprocessedEntitiesPage } from '@backstage/plugin-catalog-unproc
 />;
 ```
 
+## Customization
+
+If you want to use the provided endpoints in a different way, you can use the ApiRef doing the following:
+
+```typescript
+import { catalogUnprocessedEntitiesApiRef } from '@backstage/plugin-catalog-unprocessed-entities';
+import { useApi } from '@backstage/core-plugin-api';
+
+const catalogUnprocessedEntitiesApi = useApi(catalogUnprocessedEntitiesApiRef);
+```
+
+Note that you will need to add the API implementation to avoid your instance to crash due to "missing implementation for apiRef". To do so, add the following lines:
```

Comment:
I'm not sure that this is necessary, you'll just need to make sure that the page is rendered in your own `App.tsx` or import the plugin in this file: https://github.com/backstage/backstage/blob/99978bb0ab0175047340af9453e86cd9004af8e5/packages/app/src/plugins.ts#L20-L21

Reply:
In our case we are rendering the new component in the `HomePage`, so it was complaining due to missing API implementation. Let me try the solution. If that works I'll update the README file

Reply:
I've just tried that out (we were already exposing a grafana plugin and another custom plugin for analytics). Unfortunately, the problem is not solved, because we are still getting that missing API implementation error. Not sure if there's something defined in a different way for those plugins, as far as I see, everything looks mostly the same :(

Reply:
Hey @ivangonzalezacuna I updated the readme and documentation. Can you try it out to see if it works for you? I just did this locally and it worked for my by `exporting` the plugin from the `plugins` file.

Reply:
Yes! That worked. There was a different issue caused by an error in our side. Once fixed, the new documentation solved the issue.

Also, I just edited one of the commits, since there was a tiny typo